### ISSUE TITLE
Unable to use Alpine Linux repositories, after reopen but restore default.

### ISSUE THREAD

**Issue by 174BD7E8** [ish-app/ish#1518]
[According to wiki:](https://github.com/ish-app/ish/wiki/Using-Alpine-Linux-repositories)
```
iPhone-12-mini:~# echo https://dl-cdn.alpinelinux
.org/alpine/v3.12/main >> /etc/apk/repositories
iPhone-12-mini:~# echo https://dl-cdn.alpinelinux
.org/alpine/v3.12/community >> /etc/apk/repositor
ies
iPhone-12-mini:~# sed -i -e '/http:\/\/apk.ish.ap
p/d' /etc/apk/repositories 
iPhone-12-mini:~# cat /etc/apk/repositories 
https://dl-cdn.alpinelinux.org/alpine/v3.12/main
https://dl-cdn.alpinelinux.org/alpine/v3.12/community
```
When I close ish and reopen it:
```
iPhone-12-mini:~# cat /etc/apk/repositories 
http://apk.ish.app/v3.12-2021-06-25/main
http://apk.ish.app/v3.12-2021-06-24/community
```

**Comment by dCr01** [comment]
In /etc/profile.d nano apk_redirect.sh

echo https://dl-cdn.alpinelinux.org/alpine/v3.14/main >> /etc/apk/repositories
echo https://dl-cdn.alpinelinux.org/alpine/v3.14/community >> /etc/apk/repositories
sed -i -e '/http:\/\/apk.ish.app/d' /etc/apk/repositories

**Comment by 174BD7E8** [comment]
> In /etc/profile.d nano apk_redirect.sh
> 
> echo https://dl-cdn.alpinelinux.org/alpine/v3.14/main >> /etc/apk/repositories echo https://dl-cdn.alpinelinux.org/alpine/v3.14/community >> /etc/apk/repositories sed -i -e '/http://apk.ish.app/d' /etc/apk/repositories

Thank you!

**Comment by RyanOttignon** [comment]
I disagree with this closure - it's more of a workaround than anything.

**Comment by Rhinozz** [comment]
I disagree with this closure - it's more of a workaround than anything.

**Comment by 174BD7E8** [comment]
@Rhinozz 
Sure, i will reopen.
But i recently gave up using ish, may not pay much attention to this issue in time.

**Comment by MoriRo** [comment]
I solved it by downloading the **mini root file system x86 ** from the official download page: https://alpinelinux.org/downloads/
^^
